src/app/(layout-with-navigation)/(simulation)/tutoriel/page.tsx

```diff
@@ -21,29 +21,29 @@ export default async function Tutoriel() {
 
   return (
     <div className="flex flex-col">
-      <Title
-        data-cypress-id="tutoriel-title"
-        title={
-          <>
-            <span className="inline text-secondary">
-              <Trans>10 minutes</Trans>
-            </span>{' '}
-            <Trans>chrono pour calculer votre empreinte sur le climat</Trans>
-          </>
-        }
-      />
+      <div className="hidden md:block">
```

Comment:
Hmm je trouve ça un peu dommage de le supprimer carrément 🙍‍♂️ pour un gain d'espace dont l'intérêt est limité car on voit le bouton sur toutes les tailles d'écran mobile que j'ai testé.
Contre-proposition : couper une partie du méga block de texte (qui me parait trop long dans tous les cas) et réduire la taille du titre en `text-lg` en version mobile 

<img width="379" alt="Capture d’écran 2023-10-05 à 11 36 10" src="https://github.com/incubateur-ademe/nosgestesclimat-site-nextjs/assets/12382534/97496a1c-e8c9-4556-bcc9-a447e6de821d">

Reply:
@jeannelf

Reply:
oui ca me choque pas de le laisser ya la place

Reply:
Pour info au moins 8% de nos utilisateurs ont une hauteur d'écran qui ne leur permet pas de voir les boutons de navigation avec la version actuelle (probablement plus vu que je ne prends pas en compte l'espace de l'interface du browser). 

Et autant sur desktop je suis confiant que les gens savent scroller, autant la on a foutu un menu pile par-dessus les boutons de navigation

Reply:
Qu'est-ce que tu proposes ?

Reply:
Dans un premier temps ta solution me va très bien (peut être confirmer avec Jérémie, j'ai du mal à me rendre compte de l'importance du texte que tu as enlevé).

Et dans un second temps virer ce menu et gagner magiquement 80px de hauteur sur mobile ; mais c'est pour plus tard

(Mon commentaire était plus pour répondre à "on voit le bouton sur toutes les tailles d'écran mobile que j'ai testé.")

Reply:
J'ai peur de consulter Jérémie 😰

Reply:
Aha je lui ai envoyé un message

Reply:
on venait pas de dire en retro qu'on le contactait plus pour ce genre de trucs ? 😂

Reply:
Bah c'est lui qui a conçu cette page. Ça me semble logique qu'il ait un avis éclairé sur ce qui est le plus important à garder entre le titre ou le texte. Ça fait parti du proces d'itération sur le tutoriel

Reply:
Pas de réponse de Jérémie, la règle des 3h s'applique (3h sans réponse sur Mattermost ça veut dire qu'on peut merger ce qu'on veut)